manager/integration/tests/test_snapshot.py

```diff
@@ -220,7 +284,7 @@ def prepare_settings_for_snapshot_test(client, data_integrity, immediate_check,
     # Make the next hash time more predictable
     now = datetime.datetime.now()
     minute = (now.minute + period_in_minute) % 60
-    hour = now.hour + (now.minute + period_in_minute) / 60
+    hour = (now.hour + (now.minute + period_in_minute) / 60) % 24
```

Comment:
Fix the [failed case ](https://ci.longhorn.io/job/private/job/longhorn-tests-regression/2851/testReport/junit/tests/test_snapshot/test_snapshot_hash_global_enabled_and_per_volume_disable_and_without_immediate_hash/)

Reply:
Many thanks!